### PR TITLE
Unfollow & Like util fixes.

### PR DESCRIPTION
**Pull request by joanroig**
Unfollow util fixes, now it doesn't skip a user after 10 unfollows (https://github.com/timgrossmann/InstaPy/issues/1429).

Like util fixes:
- If a non-proper word is detected, it will not crash (solves UnicodeEncodeError: 'ascii' codec can't encode character u'\xa0' in position 7: ordinal not in range(128)) (https://github.com/timgrossmann/InstaPy/issues/1560)
- Fix when data is not properly loaded and needs a reload (adapted from https://github.com/timgrossmann/InstaPy/issues/1338 ).

### CODE REVIEW

File: instapy/unfollow_util.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def unfollow(browser,
                                        .format(int(sleep_delay/60)))
                         sleep(sleep_delay)
                         hasSlept = True
-                        continue
+                        pass
```
Comment (Tjorriemorrie) [inline instapy/unfollow_util.py:112]: instead of `pass` just delete the line.

File: instapy/unfollow_util.py [hunk 2]
```diff
@@ -360,7 +360,7 @@ def unfollow(browser,
                                     .format(int(sleep_delay/60)))
                         sleep(sleep_delay)
                         hasSlept = True
-                        continue
+                        pass
```
Comment (Tjorriemorrie) [inline instapy/unfollow_util.py:363]: same. do you mind adding a comment here as to why it is sleeping (if you can)?

**Review by Tjorriemorrie — CHANGES_REQUESTED** [review]

File: instapy/unfollow_util.py (hunk 2 above)
Comment (joanroig) [reply instapy/unfollow_util.py:363]: You mean the `logger.info('sleeping for about {}min'.format(int(sleep_delay/60)))`? Or adding a code comment?
I agree that pass statements can be deleted, I took the suggestion from https://github.com/timgrossmann/InstaPy/issues/1429 and I preferred to leave it explicit.

**Comment by timgrossmann** [comment]
@joanroig Could you please resolve the conflicts so we can merge this?

**Comment by joanroig** [comment]
@timgrossmann Resolved, tested and ready to merge.

**Comment by timgrossmann** [comment]
@joanroig Thank you very much for the contribution!
Looking forward to future ones!

**Pull request merged**